curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-css-selectors-to-style-elements.portuguese.md

```diff
@@ -81,7 +81,41 @@ tests:
 ## Solution
 <section id='solution'>
 
-```js
-// solution required
+```html
+<style>
+h2 {color: blue;}
```

Comment:
The code works, but I think we should make it more readable:
```suggestion
  h2 {
    color: blue;
  }
```

Reply:
Yeah. Thanks for the feedback!